src/sidebar/components/PressableIconButton.tsx

```diff
@@ -0,0 +1,24 @@
+import type { IconButtonProps } from '@hypothesis/frontend-shared';
+import { IconButton } from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+
+export type PressableIconButtonProps = IconButtonProps;
+
+/**
+ * An IconButton which can be used as a toggle with pressed and unpressed states
```

Comment:
```suggestion
 * An IconButton which can be used as a toggle with a more visible pressed state. Appropriate when the pressed state is not otherwise obvious from the context.
```